### ISSUE TITLE
`after` macro depends on `before`

### ISSUE THREAD

**Issue by humb1t** [bbqsrc/cucumber-rust#59]
If you want to use `after` macro it will complain about absence of `before` macro:
```
499 | / after!(an_after_fn => |_scenario| {
500 | |     thread::sleep(Duration::from_millis(world.block_build_time));
501 | | });
    | |___^
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0425]: cannot find value `an_after_fn` in this scope
   --> iroha_client/tests/cucumber.rs:516:9
    |
516 |         an_after_fn
    |         ^^^^^^^^^^^ not found in this scope

error: aborting due to 2 previous errors
```

WA: add `before` to import

**Comment by bbqsrc** [comment]
Mm, I really need to get the current dev branch into production. :smile:

The macro-based approach didn't evolve how I thought it would, so is largely removed in the development branch. As such, I won't be fixing this bug as the entire thing is being removed.

It is being replaced with a type-safe builder that is much more idiomatic Rust.

**Comment by humb1t** [comment]
@bbqsrc hi again, it was a long time ago I touched this repo)
I was busy migrating from Java to Rust developer and now I will try to complete contribution I planned to help you with "async" branch and other stuff.

I will submit more issues, but I also will try to work on them)

**Comment by bbqsrc** [comment]
Fixed in v0.6.9 published to crates.io, due to how simple it was to fix and that it's kinda  embarrassing I forgot to add `$crate::`. :smile:

I would however advise you to look at the develop branch as that macro will no longer be supported soon.
